src/modules-by-example.adoc

```diff
@@ -407,77 +403,85 @@ a generalization of Ion 1.0's shared symbol tables. As discussed in
 
 TIP: In Ion 1.1, a shared symbol table _is_ a shared module.
 
-NOTE: We intend to propose a new schema for shared modules, akin to the new `$ion_encoding` schema.
+[.line-through]##NOTE: We intend to propose a new schema for shared modules, akin to the new `$ion_encoding` schema.
 That should be easier to explain and understand than the format below.
+##
 
-For backwards compatibility purposes, shared modules are expressed using the legacy schema for
-shared symbol tables, adding a `module` field to hold macro definitions:
+[.line-through]#For backwards compatibility purposes, shared modules are expressed using the legacy schema for
+shared symbol tables, adding a `module` field to hold macro definitions:#
 
 [{nrm}]
 ----
 *$ion_1_0*
-**$ion_shared_symbol_table**::{
-  name: "com.example.graphics.3d",
-  version: 1,
-  symbols: ["x", "y", "z"],
-
-  // Schema addition follows:
-  module: $ion_1_1::(
-    (*macro* point [(int x), (int y), (int z)]
+**$ion_shared_module**::$ion_1_1::(
+  (*catalog_key* "com.example.graphics.3d" 1)
+  (*symbol_table* ["x", "y", "z"])
+  (*macro_table*
+    (*macro* point [(x *int!*), (y *int!*), (z *int!*)]
       {x: x, y: y, z: z})
-    (*macro* line  [(point a), (point b)]
+    (*macro* line  [(a *point!*), (b *point!*)]
       [a, b])
-    (*macro* poly  [(point first), (point... rest)]
-      [first, rest])
-  )
-}
+    (*macro* poly  [(first *point!*), (second *point!*), (rest *point\...+*)]
+      [first, second, rest]))
+)
 ----
 
-The `module` field here is very similar to the `*module*` S-expression inside `$ion_encoding`.
-Here, no symbolic name is declared, since one will be assigned when the module is ``*import*``ed.
+This S-expression is very similar to the `*module*` S-expression inside `$ion_encoding`.
```

Comment:
We probably have to search replace "S-expression" to "S-Expression".  I think that is the preferred spelling, yes?

Reply:
Traditional convention is "S-expression" with upper S, lower e.

* https://amazon-ion.github.io/ion-docs/docs/spec.html#sexp
* https://en.wikipedia.org/wiki/S-expression